changelog/issue-5728.md

```diff
@@ -0,0 +1,6 @@
+audience: general
+level: patch
+reference: issue 5728
+---
+
+Adds auto release lock functionality to queued locks to prevent some handlers to run forever and keep the queue locked.
```

Comment:
```suggestion
Adds auto release lock functionality to queued locks to prevent some GitHub handlers to run forever and keep the queue locked.
```
Just thought this could be a little bit more specific.

Reply:
Thanks!